lib/vmstatus/list.rb

```diff
@@ -38,26 +39,36 @@ def output(results)
                 :red
               end
 
-      hostname = vm.hostname.length > 15 ? (vm.hostname[0..12] + "...") : vm.hostname
-      on = vm.on? ? 'on' : 'off'
-      running = vm.running? ? '*' : '!'
-
-      checkout = vm.checkout ? vm.checkout.to_s : ' '
-      ttl = vm.ttl ? ("%8.2fh" % vm.ttl) : 'never'
-      user = vm.user ? vm.user : ' '
+      hostname = vm.hostname.length > 24 ? (vm.hostname[0..21] + "...") : vm.hostname
```

Comment:
why are we truncating hostnames and why change the current value?

Reply:
good point some VMs have longer names when created manually